container/karaf/src/main/filtered-resources/etc/custom.properties

```diff
@@ -39,6 +39,7 @@ org.osgi.framework.system.packages.extra=org.apache.karaf.branding,\
         sun.net.dns,\
         sun.net.spi.nameservice,\
         sun.reflect,\
+        sun.security.x509,\
```

Comment:
You will probably have to do this in the minion and sentinel versions of `custom.properties` as well. (`features/container/sentinel/src/main/filtered-resources/etc/custom.properties` and `features/container/minion/src/main/filtered-resources/etc/custom.properties`)

Reply:
Someday we should probably rework this stuff to be a template so we don't have to keep doing all the same things in 3 different places...

Reply:
done